plugins/foreman_virt_who_configure/0.1/index.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: plugin
+title: Foreman Virt Who Configure 0.1 Manual
+version: 0.1
+---
+
+# 1. {{ page.title }}
+
+This plugin make [virt-who](https://github.com/virt-who/virt-who) configuration easy. It provides simple UI for obtaining required information about the desired virt-who configuration, such as hypervisor credentials, check interval and similar. After the information is gathered, it provides a configuration script that is used to install and configure the virt-who instance. All incoming reports from such virt-who instance are tracked and monitored by the plugin.  
+
+# 2. Installation
+
+This plugins requires [Katello](https://theforeman.org/plugins/katello/index.html) to be installed in your Foreman instance. If you have it, simply install a package with the plugin (rpm only), run migrations and seed by running
+
+    yum install rubygem-foreman_virt_who_configure
+    foreman-rake db:migrate
+    foreman-rake db:seed
+
+Plugin package lives in [Katello repositories](https://fedorapeople.org/groups/katello/releases/yum/).
+
+If installed foreman from git, you can simply add foreman-virt_who_configure gem as one of the dependencies. Don't forget to run migrations and seed afterwards and obviously restart the Foreman. You still need to have Katello installed.
+
+Note for Satellite 6 users, the plugin is installed out of the box and no extra action is required.  
```

Comment:
I'll ask Katello guys if they would accept this, but it sounds to me like making have foreman_discovery as a dependency of Foreman...